shell.client/src/main/java/org/metaborg/spoofax/shell/client/Window.java

```diff
@@ -0,0 +1,59 @@
+package org.metaborg.spoofax.shell.client;
+
+import java.awt.Color;
+import java.awt.FlowLayout;
+
+import javax.swing.JFrame;
+import javax.swing.JTextPane;
+import javax.swing.text.Style;
+import javax.swing.text.StyleConstants;
+import javax.swing.text.StyledDocument;
+
+import org.metaborg.spoofax.shell.commands.StyledText;
+
+/**
+ * Represents a window.
+ */
+public class Window extends JFrame implements IDisplay {
```

Comment:
Maybe stash this and the commented out binding in ReplModule in a separate branch, or locally? As it's just an experiment.